install/kubernetes/helm/README.md

```diff
@@ -1,3 +1,3 @@
 # Installation using Helm
 
-Please follow the installation instructions from [istio.io](https://preliminary.istio.io/docs/setup/kubernetes/helm-install.html).
+Please follow the installation instructions from [istio.io](https://istio.io/docs/setup/kubernetes/helm-install.html).
```

Comment:
For master branch, shall we use `preliminary`?

Reply:
I'd rather not, since it would probably end up being copied over as-is when a release branch is created.

In general, folks playing with master must realize that their experience will not necessarily match what is on istio.io.

Reply:
make sense, thanks @geeknoid